backend/src/errors/auth.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * This file contains the class of errors related to user authentication
+ */
+
+import { CustomError } from "./errors";
+
+const DECODE_ERROR = "Error decoding the auth token. Make sure the auth token is valid";
+const TOKEN_NOT_IN_HEADER =
+  "Token was not found in the header. Be sure to use Bearer <Token> syntax";
+const INVALID_AUTH_TOKEN = "Token was invalid. Be sure to refresh token if needed.";
+
+export class AuthError extends CustomError {
+  static DECODE_ERROR = new AuthError(0, 401, DECODE_ERROR);
```

Comment:
Can you also update these error status on the API Sheet?
https://docs.google.com/spreadsheets/d/1h1LmmF9NtEkucmPMpy8WkK-ubjwUXb_qWc0b0USPN04/edit#gid=0